src/legacy/core_plugins/kibana/public/visualize/embeddable/visualize_embeddable.ts

```diff
@@ -313,6 +313,7 @@ export class VisualizeEmbeddable extends Embeddable<VisualizeInput, VisualizeOut
 
     div.setAttribute('data-test-subj', 'visualizationLoader');
     div.setAttribute('data-shared-item', '');
+    div.setAttribute('data-shared-item-container', '');
```

Comment:
Hi, this is still not right:
- the spelling of the attribute should be `data-shared-items-container ` with an S after "items"
- this part of the code makes the container surround every visualization, even when on a dashboard page. So now the dashboard page will have more than one sharable container, which causes a different error.

Reply:
Thank you! Please take a look: https://github.com/elastic/kibana/pull/52370